stable/dask/README.md

```diff
@@ -50,7 +50,7 @@ The following tables list the configurable parameters of the Dask chart and thei
 | -----------------------      | ---------------------------------| ---------------|
```

Comment:
Should  scheduler.imageTag be updated in the readme as well?

Reply:
Looking again I think you are right. I've updated the default. Thanks @davidkarlsen.